### DISCUSSION TITLE
Summer time

### DISCUSSION THREAD

**Discussion by CGenie** [FossifyOrg/General-Discussion#635] [General]
Hello,
I'm at work currently and don't have much time to correctly fill in all github bug details. However, the issue I'm experiencing is this: we had summer time change lately. I set up my alarms before that. Now they fire 1 hour later than they should (i.e. an alarm set at 6 AM fires at 7 AM now, even though it still displays as 6 AM).

**Comment by Aga-C** [comment]
Please raise a bug report for it.

**Reply by CGenie** [reply]
OK created https://github.com/FossifyOrg/Clock/issues/61
